kratos/tests/test_processes.py

```diff
@@ -755,6 +755,9 @@ def test_point_output_process_node(self):
 
         SolutionLoopPointOutputProcesses(model_part, settings, end_time, delta_time)
 
+        os.rmdir("test_parent_folder/test_subfolder")

```

Comment:
This only works if the directory is empty, which is not the case for you

-> use kratos_utilities.DeleteDirectoryIfExisting

Reply:
After the test I think it will be empty, as the result file will be removed once checked. However the kratos_utilities.DeleteDirectoryIfExisting seems to be more consistent and robust, so will modify the code.